package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "italia-app",
   "version": "2.31.0-rc.0",
-  "io_backend_api": "https://raw.githubusercontent.com/pagopa/io-backend/v10.5.0-RELEASE/api_backend.yaml",
+  "io_backend_api": "https://raw.githubusercontent.com/pagopa/io-backend/31b211f2f9486090b6bf657e7d12b8db2996ee68/api_backend.yaml",
```

Comment:
why not use tag release instead commit?

Reply:
I suppose it's not already deployed. :-D

Reply:
Exactly, we are waiting for the release :)